### PR TITLE
Test refactoring

### PR DESCRIPTION
**Pull request by taphill**
PR for test refactoring changes

- Add more robust testing more stat_tracker object
-Create empty csv fixture files for use when one isn't needed
-Create fixture files to speed up testing of league statistics methods

### CODE REVIEW

**Comment by smcunning** [comment]
Looks good to go!

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -1,41 +1,97 @@
...
+
     locations = {
       games: game_path,
       teams: team_path,
       game_teams: game_teams_path
     }
     stat_tracker = StatTracker.from_csv(locations)
 
-    assert_instance_of StatTracker, stat_tracker
+    assert_equal '2012030221', stat_tracker.game_teams[0]['game_id']
```
Comment (Stearnzy) [inline test/stat_tracker_test.rb:77]: I like how you refactored to include all points of data!

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -105,13 +161,12 @@ def test_it_knows_tie_percentage
...
-#
-#---------------LeagueStatisticsTests
 
+#---------------LeagueStatisticsTests
   def test_it_can_count_of_teams
     game_path = './fixture/games_count_teams.csv'
-    team_path = './fixture/teams.csv'
-    game_teams_path = './data/game_teams.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './fixture/game_teams_empty.csv'
```
Comment (Stearnzy) [inline test/stat_tracker_test.rb:229]: We may have to keep in mind that we're using an empty dummy here. Depending on mocks and stubs usage (not sure how that works yet), we would have to be mindful that not all methods use the same dummies.

**Review by Stearnzy — COMMENTED** [review]
left a comment on empty fixtures

**Pull request merged**